config/routes.rb

```diff
@@ -1,7 +1,8 @@
 Rails.application.routes.draw do
-  resources :clients
+  devise_for :users, path: "/", path_names: {sign_in: "login", sign_out: "logout", registration: "users"}
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root "accounts#show"
+  resources :clients
+  resource :account
```

Comment:
we're gonna allow users to switch accounts, so this should be `resources :accounts`